menas/src/test/scala/za/co/absa/enceladus/menas/integration/controllers/SchemaApiIntegrationSuite.scala

```diff
@@ -33,7 +43,20 @@ import scala.collection.immutable.HashMap
 
 @RunWith(classOf[SpringRunner])
 @SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT)
-class SchemaApiIntegrationSuite extends BaseRestApiTest {
+class SchemaApiIntegrationSuite extends BaseRestApiTest with BeforeAndAfterAll {
+
+  private val port = 8877
+  private val wireMockServer = new WireMockServer(WireMockConfiguration.wireMockConfig().port(port))
```

Comment:
👍  This is a really nice way for integration testing something that relies on queries to an external web server.